back/index.js

```diff
@@ -101,31 +105,33 @@ app.post('/api/admins', (request, response) => {
   });
 });
 
-const validUsername = 'Cindie';
-const validPassword = 'jaimelecode';
-
 app.post('/api/admins/login', (request, response) => {
-
-  const formData = request.body;
-  
-  if (formData.loginAdmin === validUsername && formData.passwordAdmin === validPassword) {
```

Comment:
are you sure you want to remove this test? without it, your login will always pass, even with wrong credentials

Reply:
Yes i'm sure because I want to check now if the password and the username exist in the db.